src/NativeBridge/dllmain.cpp

```diff
@@ -96,7 +97,11 @@ bp::dict pxCall(bp::dict& params)
         if (params.has_key(PARAM_SEED))
             seed = bp::extract<int>(params[PARAM_SEED]);
 
-        EnvironmentBlock env(i_verbose, 0, seed, s_pythonPath.c_str());
+		int maxSlots = -1;
```

Comment:
The indentation is confusing here. Have you turned off the use spaces for tabs option in your editor?